HelpSource/Classes/BelaScopeUGen.schelp

```diff
@@ -8,10 +8,10 @@ This UGen only works on Bela
 ::
 This UGen is used internally to plot signals to Bela's Oscilloscope. Direct usage of this UGen should never be needed. See link::Classes/BelaScope::
 
+BelaScopeUGen allocates the internal buffer where link::Classes/BelaScopeOut:: UGens can write data to be scoped. Only one instance of BelaScopeUGen is allowed to be operational at the same time,
+ exceeding instances will be inactive, and won't become active even if they later become the only instance present.
+If, at creation time, a BelaScopeUGen detects other already present instances, it will set its Done flag and consequently free its enclosing synth.
```

Comment:
Is this the best behaviour? One would have synths not playing (important!) only because the (relatively unimportant) scope logging feature cannot be achieved. On the other hand, it's perhaps good that something clearly stops working, so you won't be left wondering "I can hear the synth, why can't I see the scope?"

Not sure how to best do this!

Reply:
In this case, the synth to be freed is the one created by BelaScope (sclang), which is the only one which contains a BelaScopeUGen. Synths calling .belaScope() wouldn't be freed. "I can hear the synth, but not seeing the scope" is exactly what we get :)

Reply:
great